spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/StreamDeploymentsDocumentation.java

```diff
@@ -202,13 +196,11 @@ public void rollback() throws Exception {
 		this.mockMvc.perform(
 				post("/streams/deployments/rollback/{name}/{version}", "timelog1", 1)
 						.contentType(MediaType.APPLICATION_JSON))
-				.andDo(print())
 				.andExpect(status().isCreated())
 				.andDo(this.documentationHandler.document(
 						pathParameters(parameterWithName("name")
 										.description("The name of an existing stream definition (required)"),
 								parameterWithName("version").description("The version to rollback to"))));
-		Thread.sleep(30000);
```

Comment:
Ditto https://github.com/spring-cloud/spring-cloud-dataflow/pull/5349/files#r1197992799

Reply:
Not sure why the thread.sleeps were necessary.  The skipper calls were mocked so, there should be no delay in awaiting for a response.   And on tests on CI and local there was no impact for removing these thread.sleep(s).